test/integration/chain/ChainClient.spec.ts

```diff
@@ -115,16 +116,16 @@ export const ltcKeys = ECPair.fromWIF('cPnCzwHtVipcX7hb72mkkMMf4MSrknHbRnff2LDut
 export const ltcAddress = 'mysNm7METD1JffsC4E1ZW7EcPapmVm9AK4';
 
 const host = process.platform === 'win32' ? '192.168.99.100' : 'localhost';
-
-export const btcdClient = new ChainClient({
+const logger = new Logger('./test', 'debug');
```

Comment:
There is a `Logger` I created for that purpose: [it can be found in the Logger class](https://github.com/dopetard/xchange/pull/85/files#diff-2623c35db8e74a0afa56d6d9b5a64810R7)

Please use it for the tests since we don't want them to print something to a file or even the console